config/pipelines/detect_burnin.pipe

```diff
@@ -0,0 +1,105 @@
+# =============================================================================
+process detect_burnin__in
```

Comment:
Is this double underscore intentional?

Reply:
It seems to be that is the pattern here.  Is this the normal naming convention for sprokit processes?

Reply:
Please see (and consider replying to) my first comment.

Process names need to be unique. Thus, simply naming the internal processes e.g. "filter" seems collision prone. What we need are namespaced processes, but we have a very limited character set available for naming. Thus, I went with `__` as the "namespace separator". If you have a better idea, however...

I don't know that this pattern (designing a `.pipe` to be `include`d) has been used before...

Reply:
Thanks,  I understand now.  This is fine.  We may want to formalize this convention in docs at some point later, if it sticks.